mongodb/Module.md

```diff
@@ -1,37 +1,27 @@
-# Ballerina MongoDB Module
-
-Connects to MongoDB from ballerina 
-
-## Module Overview
-
+## Overview
 The Mongo DB connector allows you to connect to a Mongo DB from Ballerina and perform various operations such as `getDatabaseNames`, `getCollectionNames`, `count`, `listIndices`, `find`, `insert`, `update`, and `delete`.
 
-## Prerequisites
+This module supports Ballerina Swan Lake Beta2 version.
+
+## Configuring connector
+### Prerequisites
 
 * A mongodb with username and password
 
 * Java 11 Installed <br/> Java Development Kit (JDK) with version 11 is required.
 
-* Ballerina SLAlpha5 Installed <br/> Ballerina Swan Lake Alpha 5 is required.
+* Ballerina Swan Lake Beta2 Installed <br/> Ballerina Swan Lake Beta2 is required.
 
-## Compatibility
+## Quickstart
 
-|                             |       Version               |
-|:---------------------------:|:---------------------------:|
-| Ballerina Language          | Swan Lake Alpha 5           |
-| Mongo DB                    | V4.2.0                      |
+### Insert a document
 
-
-## Quickstart(s)
-
-## Insert a document
-
-### Step 1: Import the Mongo DB module
+#### Step 1: Import the Mongo DB module
```

Comment:
Import the MongoDB module

Reply:
Fixed by 89976637b8fed8c9a16842370da109f71b035569.